components/admin_console/system_users/index.js

```diff
@@ -70,6 +70,8 @@ function mapDispatchToProps(dispatch) {
             getUserAccessToken,
             loadProfilesAndTeamMembers,
             setSystemUsersSearch,
+            loadProfilesWithoutTeam,
+            loadProfiles: getProfiles,
```

Comment:
I'd recommend to use same name as `getProfiles` for easy search of redux actions.

```suggestion
            getProfiles,
```

Reply:
I have applied recommendations